src/ccnl-fwd/src/ccnl-echo.c

```diff
@@ -71,8 +71,6 @@ ccnl_echo_request(struct ccnl_relay_s *relay, struct ccnl_face_s *inface,
     ucp = reply->data;
     len = reply->datalen;
 
-    while (!ccnl_switch_dehead(&ucp, &len, &enc)); // for iot2014 encoding
```

Comment:
@blacksheeep I am not very familiar with this program. Is it safe to remove this line here (despite the comment saying it's just for iottlv, maybe the comment is wrong?)

Reply:
Neither me, and we have a lack of documentation.

I think this functionality is only required to distinguish CISTLV and IOTTLV, so that it can be removed.